Tensile/KernelWriterAssembly.py

```diff
@@ -6559,6 +6631,10 @@ def globalWriteBatch(self, kernel, beta, edge, lsu, atomic, gwvw, atomicW, \
             else:
               kStr += self.chooseGlobalLoad(useBuffer, bps, sumIdx, \
                         addr0, addr1, soffset=0, offset=0, extraFields="", hi16=0)
+          elif kernel["ProblemType"]["DataType"].isInt8x4():
+            # assume this is the same as for isSingle
+            kStr += self.chooseGlobalLoad(useBuffer, bps, sumIdx, \
+                      addr0, addr1, soffset=0, offset=0, extraFields="")
```

Comment:
Could make this an "OR" since the code is same on both Int8x4 and Single?

Reply:
Changed to use OR